CHANGELOG.asciidoc

```diff
@@ -31,6 +31,60 @@ https://github.com/elastic/beats/compare/v6.5.0...6.x[Check the HEAD diff]
 
 - Fix issue preventing diskio metrics collection for idle disks. {issue}9124[9124] {pull}9125[9125]
 - Allow beats to blacklist certain part of the configuration while using Central Management. {pull}9099[9099]
+*Packetbeat*
+
+*Winlogbeat*
+
+*Functionbeat*
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Enforce validation for the Central Management access token. {issue}9621[9621]
```

Comment:
Sounds like this cahngelog contains quite a bit more then expected.

Reply:
Yep, argh. I'll switch to changelog.next . Changelog merges never work right